test/user.test.js

```diff
@@ -0,0 +1,52 @@
+import { get, post, put, del, config } from '../framework/services/apiService'
+
+describe('Auth', () => {
+  it('Register', async () => {
+    const response = await post('v2/user', {
+      id: config.userid,
+      username: config.username,
+      password: config.password,
+      userStatus: 0
+    })
+
+    expect(response).toHaveProperty('statusCode', 200)
```

Comment:
у `got` нет поля `statusCode` в ответе, но есть поле `code`

```suggestion
expect(response.code).toBe(200)
```